ui/src/widgets/ui-chart/UIChart.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-    <canvas ref="chart" :class="className" />
+    <div>
+        <canvas ref="chart" :class="className" />
+        <div v-if="radialChart && !hasData" class="nrdb-ui-chart-placeholder">No Data Received</div>
```

Comment:
```suggestion
        <div v-if="radialChart && !hasData" class="nrdb-ui-chart-placeholder">No Data</div>
```

When I send an empty array to clear the data, it would show "No data Received" was misleading